server/index.js

```diff
@@ -2,9 +2,17 @@ import express from 'express';
 
 const app = express();
 
-app.use(express.static('public'));
+// app.use(express.static('public'));
 
-app.listen(3000 || process.env.PORT, () => {
-  console.log(`App listening on PORT ${3000 || process.env.PORT}`)
-});
+app.get('/', (request, response) => {
```

Comment:
I agree this is for testing, remember we are using graphQL. No endpoint like this

Reply:
Yea. It was just for for testing purposes.